fennel/datasets/test_dataset.py

```diff
@@ -1489,7 +1491,97 @@ class B:
         @inputs(A)
         def from_a(cls, a: Dataset):
             x = a.rename({"a1": "b1"})
-            return x.drop(["a2", "a3"])
+            return x.select(["b1"])
```

Comment:
Having to write a list in `select(['b1', 'b2'])` is a sub-optimal experience. Why not use *args so that user could write `select('b1', 'b2')`? 

We could also do this for drop too (maybe do *args but if there is only one element and that is list of strings, change args to be that list?)

Reply:
Added the change for ``select``. 
drop uses the ``columns=[..]`` in lots of places.
Ex:
```py
        return dealer.transform(
            lambda df: df.drop(columns=["address", "cars"]),
            schema,
        )
```

Reply:
we could support both ?